examples/kubernetesversions/kubernetes1.7.1.json

```diff
@@ -1,36 +0,0 @@
-{

```

Comment:
Does it no longer make sense to have a kubernetes 1.6.8 specific json file in examples?